Makefile

```diff
@@ -0,0 +1,24 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: csv-generator
+
+csv-generator:
+	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o build/_output/csv-generator tools/csv-generator.go
+
+clean:
+	GO111MODULE=on; \
+	go mod tidy; \
+	go mod vendor; \
+	rm -rf build/_output
+
+build: clean operator csv-generator
```

Comment:
where is operator specified in the Makefile?

Reply:
It isn't!  The current sequence only uses the Makefile to build the csv-generator. What I was saying on slack is we could expand it (in another patch perhaps) to do both. Right now in the README I changed the docs so that you run 'make' before you run the 'operator-sdk build' command.